resources/lang/de/command/messages.php

```diff
@@ -5,80 +5,80 @@
         'ask_password_help' => 'Das Passwort muss Zahlen, Groß- und Kleinbuchstaben enthalten und mindestens 8 Zeichen lang sein.',
         '2fa_help_text' => [
-            'This command will disable 2-factor authentication for a user\'s account if it is enabled. This should only be used as an account recovery command if the user is locked out of their account.',
-            'If this is not what you wanted to do, press CTRL+C to exit this process.',
+            'Dieser Befehl deaktiviert die 2-Faktor-Authentifizierung für das Konto eines Benutzers, wenn sie aktiviert ist. Dies sollte nur dann als Befehl zur Wiederherstellung eines Kontos verwendet werden, wenn der Benutzer von seinem Konto ausgeschlossen ist.',
+            'Wenn das nicht das ist was du tun wolltest, drücke Strg+C um den Prozess zu beenden.',
         ],
-        '2fa_disabled' => '2-Factor authentication wurde für :email deaktivier.',
+        '2fa_disabled' => '2-Factor Authentifizierung wurde für :email deaktiviert.',
     ],
     'schedule' => [
-        'output_line' => 'Dispatching job for first task in `:schedule` (:hash).',
+        'output_line' => 'Einplanungsjob für die erste Aufgabe in `:schedule` (:hash).',
     ],
     'maintenance' => [
-        'deleting_service_backup' => 'Deleting service backup file :file.',
+        'deleting_service_backup' => 'Lösche Service Backup Datei :file.',
     ],
     'server' => [
-        'rebuild_failed' => 'Rebuild request for ":name" (#:id) on node ":node" failed with error: :message',
+        'rebuild_failed' => 'Neuaufbauanforderung für ":name" (#:id) auf Node ":node" ist gescheitert mit Fehler: :message',
     ],
     'environment' => [
         'mail' => [
-            'ask_smtp_host' => 'SMTP Host (e.g. smtp.google.com)',
+            'ask_smtp_host' => 'SMTP Host (z.B. smtp.google.com)',
             'ask_smtp_port' => 'SMTP Port',
-            'ask_smtp_username' => 'SMTP Username',
-            'ask_smtp_password' => 'SMTP Password',
+            'ask_smtp_username' => 'SMTP Benutzername',
+            'ask_smtp_password' => 'SMTP Passwort',
             'ask_mailgun_domain' => 'Mailgun Domain',
             'ask_mailgun_secret' => 'Mailgun Secret',
             'ask_mandrill_secret' => 'Mandrill Secret',
             'ask_postmark_username' => 'Postmark API Key',
-            'ask_driver' => 'Which driver should be used for sending emails?',
-            'ask_mail_from' => 'Email address emails should originate from',
-            'ask_mail_name' => 'Name that emails should appear from',
-            'ask_encryption' => 'Encryption method to use',
+            'ask_driver' => 'Welcher Treiber soll für das Senden von E-Mails verwendet werden?',
+            'ask_mail_from' => 'E-Mail Adressen E-Mails ausgehen von',
+            'ask_mail_name' => 'Name von welchem E-Mails erscheinen sollen',
+            'ask_encryption' => 'Zu benutzende Verschlüsselungsmethode',
```

Comment:
"benutzen" / "benutzende" is fine, but I think "verwenden" sounds nicer ("Zu verwendende...")